python-data-version-control/src/evaluate.py

```diff
@@ -0,0 +1,29 @@
+from joblib import load
+import json
+from pathlib import Path
+import numpy as np
+import pandas as pd
+from sklearn.metrics import accuracy_score
+
+from train import load_images, load_labels, preprocess
+
+
+def main(repo_path):
+    test_csv_path = repo_path / "data/prepared/test.csv"
+    test_df = pd.read_csv(test_csv_path)
+    raw_images = load_images(data_frame=test_df, column_name="filename")
+    labels = load_labels(data_frame=test_df, column_name="label")
+    preprocessed = [preprocess(image) for image in raw_images]
+    test_data = np.concatenate(preprocessed, axis=0)
+    model_path = repo_path / "model"
+    model = load(model_path / "model.joblib")
```

Comment:
Splitting this across two lines doesn't feel super necessary to me, especially since you don't use `model_path` a second time:

```suggestion
    model = load(repo_path / "model/model.joblib")
```